subgraph/generated/schema.ts

```diff
@@ -461,27 +458,27 @@ export class Extend extends Entity {
   }
 }
 
-export class ActiveSubscription extends Entity {
+export class Subscription extends Entity {
```

Comment:
All right, @cmwhited @Theodus, we need to rename this, because it conflicts with GraphQL's `Subscription` root type now.

Reply:
I guess we have two choices

1. `UserSubscription`
2. `Sub`